celery_worker/worker.py

```
@@ -1,59 +1,109 @@
+        response = requests.post(AGENT_URL, json=payload, timeout=600)
+
+        # Логируем сырой ответ для отладки
+        logger.info(f"Ответ Агента (код {response.status_code}): {response.text}")
+
+        response.raise_for_status()
         result = response.json()
+
+        if result.get("success") or "answer" in result:
+            status = "completed"
+            answer = result.get("answer")
+            # Если агент сохранил новую картинку (с разметкой), он должен вернуть 'processed_path'
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF003)